### DISCUSSION TITLE
Online Flasher falls keine IDE vorhanden ist.

### DISCUSSION THREAD

**Discussion by beabel** [beabel/radiowecker#21] [Ankündigungen / Announcements]
Ich habe mal zum schnellen flashen dies zur Verfügung gestellt:
### [Radiowecker Web Flash](https://ne-xt.de/esp_flasher/)

Update zu Version v4.0.3

Screenshot:
<img width="914" height="689" alt="image" src="https://github.com/user-attachments/assets/15a5aa65-8dd8-4120-8a71-05fc2e4411da" />

**Comment by DarnokHillstar** [comment]
Klappt nun perfekt! Danke für die Hilfe gestern.
